angular/projects/global/src/lib/services/search.service.ts

```diff
@@ -100,6 +101,11 @@ export class SearchService {
         });
       });
     }
+    if (_in && _in !== {}) {
```

Comment:
This will always pass because `_in` will always not equal `{}`.  `{} === {}` will always be false.  If checking for any empty object this way it will need to be converted to a string first or another way will need to be used.

`JSON.stringify(_in) !== '{}'`  or
`Object.keys(_in).length !== 0`

Reply:
Good catch! Fixed in d2bcc9b